lib/bundler/fetcher.rb

```diff
@@ -53,6 +56,10 @@ def initialize(remote_uri)
     class << self
       attr_accessor :disable_endpoint, :api_timeout, :redirect_limit, :max_retries
 
+      Fetcher.redirect_limit = 5  # How many redirects to allow in one request
```

Comment:
Pretty sure this can just be `self.` instead of `Fetcher.`. Also, lolololololo that these were class-lever attr_accessors that we didn't use.